src/SFA.DAS.AANHub.Application.UnitTests/Partners/Queries/GetPartnerMemberResultTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Partners.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Partners.Queries
+{
+    [TestFixture]
+    public class GetPartnerMemberResultTest
+    {
+        [Test, AutoMoqData]
+        public void Partner_PopulatesGetPartnerMemberResultFromPartner(Partner partner)
+        {
+            var response = (GetPartnerMemberResult?)partner;
+
+            Assert.That(response, Is.Not.Null);
+
+            Assert.AreEqual(partner.MemberId, response!.MemberId);
```

Comment:
We should probably agree to whether we use FluentAssertions or NUnit's `Assert` syntax to validate tests.

Thoughts @DevenKShah ?

Reply:
I will leave this for us to look at a later stage

Reply:
I have not been consistent myself when it comes to assertion. I don't mind what you use, as both libs are reference anyway.